### PR TITLE
Add missing parameters

### PR DESCRIPTION
**Pull request by osterman**
## what
* Support build timeouts and badge URLs
* Support GitHub webhooks

## why
* Failed builds would keep retrying for 60 minutes
* Badges are nice for the README
* Trigger only only GitHub releases (optional)

### CODE REVIEW

File: docs/terraform.md [hunk 1]
```diff
@@ -84,6 +87,7 @@
 
 | Name | Description |
 |------|-------------|
+| badge_url | the url of the build badge when badge_enabled is enabled |
```
Comment (aknysh) [inline docs/terraform.md]:
```suggestion
| badge_url | The URL of the build badge when `badge_enabled` is enabled |
```

File: main.tf [hunk 2]
```diff
@@ -75,7 +75,7 @@ module "ecs_alb_service_task" {
 
 module "ecs_codepipeline" {
   enabled            = "${var.codepipeline_enabled}"
-  source             = "git::https://github.com/cloudposse/terraform-aws-ecs-codepipeline.git?ref=tags/0.2.0"
+  source             = "git::https://github.com/cloudposse/terraform-aws-ecs-codepipeline.git?ref=add-params"
```
Comment (aknysh) [inline main.tf]: pin to a release

File: outputs.tf [hunk 3]
```diff
@@ -27,3 +27,8 @@ output "service_security_group_id" {
   description = "Security Group id of the ECS task"
   value       = "${module.ecs_alb_service_task.service_security_group_id}"
 }
+
+output "badge_url" {
+  description = "the url of the build badge when badge_enabled is enabled"
```
Comment (aknysh) [inline outputs.tf]:
```suggestion
  description = "The URL of the build badge when `badge_enabled` is enabled"
```

File: main.tf [hunk 4]
```diff
@@ -74,20 +74,31 @@ module "ecs_alb_service_task" {
...
-  github_oauth_token = "${var.github_oauth_token}"
-  repo_owner         = "${var.repo_owner}"
-  repo_name          = "${var.repo_name}"
-  branch             = "${var.branch}"
-  image_repo_name    = "${module.ecr.repository_name}"
-  service_name       = "${module.ecs_alb_service_task.service_name}"
-  ecs_cluster_name   = "${var.ecs_cluster_name}"
-  privileged_mode    = "true"
+  enabled               = "${var.codepipeline_enabled}"
+  source                = "git::https://github.com/cloudposse/terraform-aws-ecs-codepipeline.git?ref=add-webhook-support"
```
Comment (joshmyers) [inline main.tf]: This branch has been merged and a release (0.4.0) cut

**Pull request merged**